ui/apps/platform/src/Containers/ComplianceEnhanced/ClusterCompliance/ScanConfigs/ViewScanConfigDetail.tsx

```diff
@@ -0,0 +1,123 @@
+                        Scan schedules
+                    </BreadcrumbItemLink>
+                    {!isLoading && !error && scanConfig && (
+                        <BreadcrumbItem isActive>{scanConfig.scanName}</BreadcrumbItem>
+                    )}
+                </Breadcrumb>
+            </PageSection>
+            <Divider component="div" />
+            <PageSection variant="light" padding={{ default: 'noPadding' }}>
+                {!isLoading && !error && scanConfig && (
+                    <Flex
+                        alignItems={{ default: 'alignItemsCenter' }}
+                        className="pf-u-py-lg pf-u-px-lg"
+                    >
+                        <FlexItem flex={{ default: 'flex_1' }}>
+                            <Title headingLevel="h1">{scanConfig.scanName}</Title>
+                        </FlexItem>
+                        {hasWriteAccessForCompliance && (
+                            <FlexItem align={{ default: 'alignRight' }}>
+                                <Button
+                                    variant="primary"
+                                    component={LinkShim}
+                                    href={`${generatePath(complianceEnhancedScanConfigDetailPath, {
+                                        scanConfigId: scanConfig.id,
+                                    })}?action=edit`}
+                                >
+                                    Edit scan schedule
+                                </Button>
+                            </FlexItem>
+                        )}
+                    </Flex>
+                )}
+            </PageSection>
+            <Divider component="div" />
+            <PageSection isCenterAligned>
+                {isLoading ? (
+                    <Bullseye>
+                        <Spinner isSVG />
+                    </Bullseye>
+                ) : (
+                    error && (
+                        <Alert
+                            variant="warning"
+                            title="Unable to fetch scan schedule"
+                            component="div"
+                            isInline
+                        >
+                            {getAxiosErrorMessage(error)}
+                        </Alert>
+                    )
+                )}
+                {!isLoading && scanConfig && (
+                    <Grid hasGutter>
+                        <GridItem sm={12} md={6}>
+                            <ScanConfigParameters scanConfig={scanConfig} />
+                        </GridItem>
+                        <GridItem sm={12} md={6}>
+                            <ScanConfigProfiles profiles={scanConfig.scanConfig.profiles} />
+                        </GridItem>
+                        <GridItem sm={12}>
+                            <ScanConfigClustersTable
+                                clusterScanStatuses={scanConfig.clusterStatus}
```

Comment:
This didn't seem to populate for me when I tested this locally. Is there something else I needed to do to get this to render in the UI?

```
 $ ./bin/roxcurl /v2/compliance/scan/configurations | jq '.configurations[].clusterStatus'
[
  {
    "clusterId": "3bbadeca-82ee-4ff5-ae0f-15a89aca3863",
    "errors": [
      ""
    ],
    "clusterName": "remote"
  }
]
[
  {
    "clusterId": "3bbadeca-82ee-4ff5-ae0f-15a89aca3863",
    "errors": [
      ""
    ],
    "clusterName": "remote"
  }
]
```

Reply:
I was expecting some kind "status" value returned from the API. Is it simply that an empty `errors` array is healthy, and a non-empty `errors` array is unhealthy?

Reply:
That seems reasonable, but I'll let @dashrews78 weigh-in on if there is another part of the ACS API dedicated to serving up the cluster health.

There is the integrations API:

```
$ ./bin/roxcurl /v2/compliance/integrations | jq .
{
  "integrations": [
    {
      "id": "8469f1f9-af2a-4ebf-bb96-28c0202f8639",
      "version": "v1.4.0",
      "clusterId": "3bbadeca-82ee-4ff5-ae0f-15a89aca3863",
      "clusterName": "remote",
      "namespace": "openshift-compliance",
      "statusErrors": []
    }
  ]
}
```

But that has a similar pattern where it looks like we infer healthy based on the absence of anything in the `statusErrors` list.

Reply:
Yeah I the presence or not of the `errors` is how status can be inferred.  I suppose we could add that to the API if you really need us to.

Reply:
Tested this with the latest commit and it worked for me locally. Thanks!